src/main/java/Data/Data.java

```diff
@@ -142,4 +146,12 @@ public String toString() {
                 ", SYNC_CLIENT_URL='" + SYNC_CLIENT_URL + '\'' +
                 '}';
     }
+
+    public String getDATABASE_URL() {
```

Comment:
Please follow the convention of this class and move these getter/setter up where the other getters and setters are.